src/app/src/app.reducers.js

```diff
@@ -28,6 +34,12 @@ const appReducer = createReducer(
                     $set: null,
                 },
             }),
+        [toggleModal]: state =>
```

Comment:
In other projects I've seen us ben more granular with actions so their result is predictable, i.e. `openModal` and `closeModal` as separate actions. That said, this app is pretty simple. Do you foresee any potential for conflict with the state of `isModalDisplayed`?

No strong opinion here, just a consideration.

Reply:
Yeah, that's a good point. I think in most cases I would do the open/close pattern. I think I did it this way because the app is so simple. But I'm going to change it to what you suggested.